client.go

```diff
@@ -49,6 +49,10 @@ type Client interface {
 	LSExtIdsAdd(ls string, external_ids map[string]string) (*OvnCommand, error)
 	// Del external_ids from logical_switch
 	LSExtIdsDel(ls string, external_ids map[string]string) (*OvnCommand, error)
+	// Add keys/vals to logical switch external_ids or other_config records
+	LSAuxAdd(ls string, auxConf map[string]string, auxColumn string) (*OvnCommand, error)
```

Comment:
It is embarrassing that we didn't have a consistent way to manipulate fields like external_ids and other_config. Now that we already had LSExtIdsAdd/LSExtIdsDel for LS external_ids, I think it may be better to avoid adding overlapping APIs just for LS.

There are two options:

Option1: We can just add LSOtherConfigAdd (in fact it is better to be named as xxxSet, which should overwrite existing key/value) and LSOtherConfigDel. (we need to fix the behavior of LSExtIdsAdd so that it takes care of existed key updating)

Option2: It may be even better to just add common APIs for these common columns to support all tables instead of separate APIs for each table. These APIs can just take "tableName" as an additional parameter, such as:

* AuxKeyValSet(table_name string, row_name string, auxColumn string, auxConf map[string]string) (*OvnCommand, error)
* AuxKeyValDel(table_name string, row_name string, auxColumn string, auxConf map[string]string) (*OvnCommand, error)

What do you think?

Reply:
@hzhou8 IMHO option2 is the best, to solve this once and for all (tables). Maybe even a 3rd method - AuxKeyValGet()?